modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -8,24 +8,29 @@ import {
   removeSQON,
 } from '../SQONView/utils';
 import './BooleanAgg.css';
+import TextHighlight from '../TextHighlight';
 
 export default ({
   field,
   buckets,
   handleValueClick = () => {},
   isActive = () => false,
+  collapsible,
+  WrapperComponent,
+  displayName,
+  searchString,
   ...rest
 }) => {
   const trueBucket = buckets.find(({ key }) => key === '1');
   const falseBucket = buckets.find(({ key }) => key === '0');
   const dotField = field.replace(/__/g, '.');
 
   const isTrueActive = isActive({
-    value: true,
+    value: 'true',
```

Comment:
what happens if these are booleans?

Reply:
seems there's a truthy check somewhere that would break it... I think using strings make it more strict

Reply:
might be work finding those checks and fixing them. It could break on other values as well :)

Reply:
so the server currently doesn't handle `false` value in sqon, so we're going to stick to using strings to abstract the values as ES is ok with querrying `false` with `"false"` :)